### ISSUE TITLE
Samples do not build

### ISSUE THREAD

**Issue by chrbsg** [microsoft/cognitive-services-speech-sdk-go#58]
- README.md does not state how to build the samples in the `samples` directory
- cd samples && go build results in errors:

```
# github.com/Microsoft/cognitive-services-speech-sdk-go/samples/speaker_recognition
speaker_recognition/independent_identification.go:12:2: imported and not used: "github.com/Microsoft/cognitive-services-speech-sdk-go/samples/helpers"
speaker_recognition/independent_identification.go:16:84: undefined: common
speaker_recognition/independent_identification.go:23:20: outcome.profile undefined (cannot refer to unexported field or method profile)
speaker_recognition/independent_identification.go:42:37: undefined: common
speaker_recognition/independent_identification.go:43:21: undefined: VoiceProfileEnrollmentResult
speaker_recognition/independent_verification.go:12:2: imported and not used: "github.com/Microsoft/cognitive-services-speech-sdk-go/samples/helpers"
speaker_recognition/independent_verification.go:16:6: GetNewVoiceProfileFromClient redeclared in this block
	/home/chrb/git/cognitive-services-speech-sdk-go/samples/speaker_recognition/independent_identification.go:16:110: previous declaration
speaker_recognition/independent_verification.go:16:84: undefined: common
speaker_recognition/independent_verification.go:41:6: EnrollProfile redeclared in this block
	/home/chrb/git/cognitive-services-speech-sdk-go/samples/speaker_recognition/independent_identification.go:41:99: previous declaration
speaker_recognition/independent_verification.go:68:6: DeleteProfile redeclared in this block
	/home/chrb/git/cognitive-services-speech-sdk-go/samples/speaker_recognition/independent_identification.go:68:65: previous declaration
speaker_recognition/independent_identification.go:43:21: too many errors

```

This is Ubuntu LTS (20.04.4) with go-1.17.4

**Comment by glharper** [comment]
@chrbsg Thanks for using the Speech SDK and writing this issue up. I checked in some fixes for this with [this commit](https://github.com/microsoft/cognitive-services-speech-sdk-go/commit/9b26a1a726d23ed346f64953f5de0d33486aeed8)

Could you pull latest and give this another try?

**Comment by chrbsg** [comment]
@glharper it fails as-is due to samples/go.mod pulling in the older version (v1.20.0) of the SDK, but if I add `replace github.com/Microsoft/cognitive-services-speech-sdk-go => ../` to samples/go.mod then it builds ok. It might make sense to build the samples against the current git checkout, rather than some tag, but you could tag it and update the go.mod too.

**Comment by jhakulin** [comment]
closed as 1.22.0 release has been done.
